client/src/main/kotlin/no/nav/su/se/bakover/client/StubClientsBuilder.kt

```diff
@@ -39,7 +38,7 @@ object StubClientsBuilder : ClientsBuilder {
 
     override fun build(applicationConfig: ApplicationConfig): Clients {
         return Clients(
-            oauth = AzureClientStub.also { log.warn("********** Using stub for ${OAuth::class.java} **********") },
+            oauth = AzureClient(applicationConfig.azure.clientId, applicationConfig.azure.clientSecret, applicationConfig.azure.wellKnownUrl),
```

Comment:
Skal denne være default?

Reply:
Ja. Vi har ikke lenger noen token-utsteding her i bakover (bortsett fra i tester, men da er det en egen ClientsBuilder som brukes).
Jeg tenker at dersom det viser seg at vi har behov for å kjøre bakover og gjøre API-kall hit uten at vi kjører oauth-serveren så får vi heller se på om vi kan trikse det til da.